code/index.html

```diff
@@ -1,22 +1,288 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Maple News | Breaking news about the sweet gold</title>
+  <link rel="icon" href="./other-images/maple-leaf.ico">
+  <link rel="stylesheet" type="text/css" href="./style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600&family=Raleway:wght@600&display=swap"
+    rel="stylesheet">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-2/css/all.min.css" rel="stylesheet">
+</head>
+
+<body>
+  <!-- START Header with logo, links and date only visible on mobile (max 1024px) -->
+  <header class="mobile">
+    <div class="mobile-wrap">
+      <input type="checkbox" id="checkbox">
+      <nav>
+        <ul>
+          <li class="news">
+            <a target="_blank" href="https://www.svt.se">News</a>
+          </li>
+          <li class="law">
+            <a target="_blank" href="https://www.wsj.com/news/business/law-legal">Law & Business</a>
+          </li>
+          <li class="travel">
+            <a target="_blank" href="https://edition.cnn.com/travel/">Travel</a>
+          </li>
+          <li class="sports">
+            <a target="_blank" href="https://www.lapresse.ca/sports/">Sports</a>
+          </li>
+          <li class="society">
+            <a target="_blank" href="https://www.healthline.com/health-news">Society</a>
+          </li>
+          <li class="weather">
+            <a target="_blank" href="https://www.svt.se/vader/">Weather</a>
+          </li>
+        </ul>
+      </nav>
+      <label class="mobile-wrap-item1" for="checkbox">
+        <i class="fa fa-bars menu-icon">
+        </i>
+      </label>
+      <a class="mobile-wrap-item2" href="https://maple-news.netlify.app">
+        <img src="./other-images/logo-maple-news.svg" class="logo-mobile" alt="Logo Maple News">
+      </a>
+      <time class="mobile-wrap-item3">
+        <!-- Show date and time, link Stack Overflow used: 
+        https://stackoverflow.com/questions/32540044/html-display-current-date/32540850#32540850 -->
```

Comment:
For education purposes you mention where you got the inspiration to some of the more advanced features, this was very helpful.